docs/use-the-network/light-hotspots/guides/Cotx-Networks.mdx

```diff
@@ -0,0 +1,61 @@
+# Cotx X3 LoRaWAN Gateway Setup
+
+This guide provides instructions on how to configure a Cotx X3 LoRaWAN gateway to work with the Helium network.
+
+:::warning
+
+Cotx X3 gateway will only mine HNT through packet transfer, and is considered a non-PoC Light Hotspot. The ability to mine HNT with anything other than [Approved Hotspots](https://helium.com/mine) is not currently supported. Please join the [#gateway-development](https://discord.gg/helium) channel in the Helium Discord Server for the latest updates.
+
+:::
+
+## Prerequisites
+
+Prior to configuring your Cotx X3 to work with the Helium network, please confirm that it is connected to the Internet and updated to the latest firmware. 
+
+- Connect to a Wi-Fi or Ethernet network with Internet access.
+- Use SSH. Your username is `admin` and the default password is `123xxxxxx`, please contact the salesperson for the password.
+- Once logged in, please change your password.
+
+## Configure LoRa Frequency Plan
+
+To be able to send and receive LoRa data, your gateway must be configured to the frequencies specified for your region. You may be able to use the predefined frequency in the Cotx X3 or some cases may need to manually set them per the frequencies used by Helium in your region.
+
+1. Under Radio Settings, select your region in the [Frequency Plan](https://docs.helium.com/lorawan-on-helium/frequency-plans) drop down and, if applicable, the Frequency Sub Band. For example, in the US, select US915 for the frequency plan and 2 for the sub band.
+2. For some regions, it my be necessary to manually populate Populate the Customized Channel Settings based on the Helium frequencies for your region.
+
+Refer to the [Helium sample LoRa packet forwarder global.conf files](https://github.com/helium/sx1302_hal/tree/helium/hotspot/packet_forwarder) for your region as a guide to the values to use. You can check the values used by the pre-defined frequency plans by viewing the global_conf.json file: `cat /root/helium/lora_net/lora_pkt_fwd/global_conf.json`
+
+## Install Helium Gateway (gateway-rs) on the Cotx X3
+
+If your supported LoRa gateway did not come with helium-gateway pre-installed, manual installation requires you to:
+
+1. Configure the packet forwarder on the gateway to forward to the helium-gateway application. This varies per gateway but the goal is to set the packet forwarder to forward to the (default) configured helium-gateway on `127.0.0.1` at udp port `1680`
+2. Connect to the Cotx X3 over SSH. 
+3. `scp` a downloaded release package for the supported platform to the gateway. Copy the Helium Gateway package to the Cotx X3. Run this command from on the computer within the directory containing the package `scp helium-gateway-<version>-<platform>.deb <gateway>:/tmp/`.
+4. Install the Helium Gateway package `dpkg --install /tmp/helium-gateway-<version>-<platform>.deb`.
```

Comment:
you are missing the actual platform name in there. Which means you will need to issue a PR against gateway-rs to create a package for it.

Reply:
> you are missing the actual platform name in there. Which means you will need to issue a PR against gateway-rs to create a package for it.

Ok, Thanks! I have successfully added the missing platform name. Is there anything else that needs to be changed?